packages/frontend/core/src/app.tsx

```diff
@@ -46,22 +48,28 @@ async function loadLanguage() {
 
 const languageLoadingPromise = loadLanguage().catch(console.error);
 
+const globalSWRConfig: SWRConfiguration = {
+  suspense: true,
+};
```

Comment:
it seems a bit dangerous to me to turn this flag on globally. Can you suggest any other side effects it will bring in?